next-docs/pages/components/tabs.tsx

```diff
@@ -39,27 +88,201 @@ const PreviewTabs = () => {
+            <TabLinkFill href="#4">Tomorrow</TabLinkFill>,
+            <TabLinkFill href="#5">Future</TabLinkFill>,
+            <TabLinkFill href="#6">Outright</TabLinkFill>,
+            <TabLinkFill href="#7">Leagues</TabLinkFill>,
+            <TabLinkFill href="#8">News</TabLinkFill>,
+          ]}
+        />
+      }
+      code={`import { Tabs, TabLinkFill } from '@heathmont/moon-core';
+
+<Tabs
+  items={[
+    <TabLinkFill href="#2">Featured</TabLinkFill>,
+    <TabLinkFill href="#3">Today</TabLinkFill>,
+    <TabLinkFill href="#4">Tomorrow</TabLinkFill>,
+    <TabLinkFill href="#5">Future</TabLinkFill>,
+    <TabLinkFill href="#6">Outright</TabLinkFill>,
+    <TabLinkFill href="#7">Leagues</TabLinkFill>,
+    <TabLinkFill href="#8">News</TabLinkFill>,
+  ]}
+/>
+`}
+    />
+  );
+};
+
+const PreviewTabsFillContainer = () => {
+  return (
+    <Preview
+      title="Segment control"
+      preview={
+        <Tabs
+          isContainer
+          items={[
+            <TabLinkFill href="#2">Featured</TabLinkFill>,
+            <TabLinkFill href="#3">Today</TabLinkFill>,
+            <TabLinkFill href="#4">Tomorrow</TabLinkFill>,
+            <TabLinkFill href="#5">Future</TabLinkFill>,
+            <TabLinkFill href="#6">Outright</TabLinkFill>,
+            <TabLinkFill href="#7">Leagues</TabLinkFill>,
+            <TabLinkFill href="#8">News</TabLinkFill>,
+          ]}
+        />
+      }
+      code={`import { Tabs, TabLinkFill } from '@heathmont/moon-core';
+
+<Tabs
+  isContainer
+  items={[
+    <TabLinkFill href="#2">Featured</TabLinkFill>,
+    <TabLinkFill href="#3">Today</TabLinkFill>,
+    <TabLinkFill href="#4">Tomorrow</TabLinkFill>,
+    <TabLinkFill href="#5">Future</TabLinkFill>,
+    <TabLinkFill href="#6">Outright</TabLinkFill>,
+    <TabLinkFill href="#7">Leagues</TabLinkFill>,
+    <TabLinkFill href="#8">News</TabLinkFill>,
+  ]}
+/>
+`}
+    />
+  );
+};
+
+export default function PageTabsNew() {
   return (
     <>
       <section className="mt-8">
         <h1 className="text-5xl font-semibold">Tabs</h1>
         <p className="text-lg mt-4">
-          A menu of items for users to move between sections of the application.
+          Use tabs to allow users to navigate easily between views within the
+          same context.
         </p>
-
         <p className="text-lg mt-4">
           By default, tabs will provide an accessible skip link, and overflow
           with horizontal scrolling.
         </p>
         <p className="text-lg mt-4">
-          TabLink component provides the tab interaction.
+          TabLink or TabLinkFill components provides the tab interaction.
+        </p>
+        <p className="text-lg mt-4">
+          There are three types of Tabs items:
+          <ol className="ml-4">
+            <li>1. Underline: top or bottom (TabLink);</li>
+            <li>2. Filled (TabLinkFill);</li>
+            <li>3. Segment control (TabLinkFill);</li>
+          </ol>
         </p>
```

Comment:
Don't think that we need that block at all

Reply:
Delete